homeassistant/components/tuya/strings.json

```diff
@@ -897,6 +897,12 @@
       },
       "sterilization": {
         "name": "Sterilization"
+      },
+      "alarm_sound": {
+        "name": "Arm/Disarm sound"
+      },
+      "alarm_light": {
+        "name": "Siren"
```

Comment:
In my mind "siren" is a sound not a light...
So this name feels weird

Reply:
Then it would be better to change the translation key => alarm_siren and add a comment in the entity description.